generic3g/OuterMetaComponent/write_restart.F90

```diff
@@ -42,17 +42,17 @@ module recursive subroutine write_restart(this, importState, exportState, clock,
       subdir = get_checkpoint_subdir(this%hconfig, currTime, _RC)
 
       if (this%component_spec%misc%checkpoint_controls%import) then
-         filename = mapl_PathJoin(subdir, this%get_name() // '_import.nc')
+         filename = mapl_PathJoin(subdir, driver%get_name() // '_import.nc')
```

Comment:
I thought "we" modified `this%get_name` so that using the driver was not necessary here.